conjure_python_client/_serde/decoder.py

```diff
@@ -99,7 +99,8 @@ def decode_conjure_union_type(cls, obj, conjure_type):
 
         deserialized = {}  # type: Dict[str, Any]
         if type_of_union not in obj or obj[type_of_union] is None:
-            cls.check_null_field(obj, deserialized, conjure_field_definition)
+            cls.check_null_field(
+                obj, deserialized, attribute, conjure_field_definition)
```

Comment:
here's the actual change I wanted to make in this PR

Reply:
For context, this got fixed on develop in https://github.com/palantir/conjure-python-client/pull/111